src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/filters/components/form/main.js

```diff
@@ -136,7 +136,7 @@ define(['config'], function(Config) {
                     options: {
                         el: $element,
                         fieldsUrl: typeConfig.fields,
-                        operatorsUrl: constants.operatorsUrl,
+                        operatorsUrl: constants.operatorsUrl + '?locale=' + AppConfig.getUser().locale,
```

Comment:
The operators are translated in all three system languages (and even some more if I can remember). In my opinion the operators need to be translated in the system language of the user, because it's weird to see everything in english but the translations of the operators are in lets say french.